Sources/Extensions/SwiftStdlib/SignedNumericExtensions.swift

```diff
@@ -22,4 +22,39 @@ public extension SignedNumeric {
 		guard let number = self as? NSNumber else { return "" }
 		return formatter.string(from: number) ?? ""
 	}
+    
+    /// SwifterSwift: Spelled out representation of a number.
+    ///
+    ///        print((12.32).spelledOutString()) // prints "twelve point three two"
+    ///
+    /// - Parameter locale: Locale, default is .current.
+    /// - Returns: String representation of number spelled in specified locale language. E.g. input 92, output in "en": "ninety-two"
+    public func spelledOutString(locale: Locale = .current) -> String? {
+        let formatter = NumberFormatter()
+        formatter.locale = locale
+        formatter.numberStyle = .spellOut
+        
+        guard let number = self as? NSNumber else { return nil }
+        return formatter.string(from: number)
+    }
+    
+    /// SwifterSwift: Ordinal representation of an integer.
+    ///
+    ///        print((12).ordinalString()) // prints "12th"
+    ///
+    /// - Parameter locale: Locale, default is .current.
+    /// - Returns: String ordinal representation of number in specified locale language. E.g. input 92, output in "en": "92nd"
+    public func ordinalString(locale: Locale = .current) -> String? {
+        let formatter = NumberFormatter()
+        formatter.locale = locale
+        
+        if #available(iOS 9.0, macOS 10.11, *) {
```

Comment:
I think it makes more sense wrap the entire method with `#available(iOS 9.0, macOS 10.11, *)`

Reply:
Also please move this extension to **SignedIntegerExtensions.swift**, as ordinal representation is only valid for integers.